From: Adrien Plazas <kekun.plazas@laposte.net>
Date: Wed, 10 Jul 2019 19:24:33 +0200
Subject: librem5: Drop the shortcuts window

The window doesn't fit on the screen and it's not very relevant.
---
 src/resources/gtk/page-menu-popover.ui | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/resources/gtk/page-menu-popover.ui b/src/resources/gtk/page-menu-popover.ui
index 6942c99..a631085 100644
--- a/src/resources/gtk/page-menu-popover.ui
+++ b/src/resources/gtk/page-menu-popover.ui
@@ -305,7 +305,7 @@
                 <property name="can_focus">True</property>
                 <property name="text" translatable="yes">_Keyboard Shortcuts</property>
                 <property name="action-name">app.shortcuts</property>
-                <property name="visible">True</property>
+                <property name="visible">False</property>
               </object>
             </child>
             <!-- FRAGILE: This button is manually removed for app mode/Pantheon in ephy-header-bar.c. -->
